backend/src/schema/types/schema.gql

```diff
@@ -32,9 +32,9 @@ type Mutation {
   # Unshout the given Type and ID
   unshout(id: ID!, type: ShoutTypeEnum): Boolean!
   # Follow the given Type and ID
-  follow(id: ID!, type: FollowTypeEnum): Boolean!
+  follow(id: ID!, type: FollowTypeEnum): User
```

Comment:
:+1: nice, looks great!

If you need any help or another review, feel free to ask for it @vbelolapotkov.

Reply:
@roschaefer actually this looks a little be weird, especially `FollowTypeEnum` argument. I suppose this was done to reuse the mutation in the future to allow following other entities like posts or something else.

In that case, I'd expect some new `Followable` type (e.g. union type) or interface is returned. What do you think?

Reply:
This makes no sense at all. I guess the `FollowTypeEnum` was supposed to include `Organization`s as well. Something that we decided to not implement any soon. @vbelolapotkov do you want to land another quick contribution and remove `FollowTypeEnum`?

Reply:
@roschaefer will be happy to help here as well. I'd rename the mutation as well to `(un)followUser`

Reply:
@roschaefer here it is #1660 , please review when you have time